daemon/api.go

```diff
@@ -967,6 +976,119 @@ func deleteCapability(c *Command, r *http.Request) Response {
 	}
 }
 
+// skillGrant is a part of response from getSkills().
+type skillGrant struct {
+	Snap string `json:"snap"`
+	Name string `json:"name"`
+}
+
+// skillInfo is a part of response from getSkills().
+type skillInfo struct {
+	Snap  string       `json:"snap"`
+	Name  string       `json:"name"`
+	Type  string       `json:"type"`
+	Label string       `json:"label"`
+	Slots []skillGrant `json:"slots"`
```

Comment:
Sorry, I've been going back and forth with the name of this field, and I think this is still wrong. Can we please replace it with `GrantedTo ... json:"granted-to"`. It feels strange to say "slots" without pointing out what the relationship is.